native/src/main/java/org/ballerinax/asb/sender/MessageSender.java

```diff
@@ -98,22 +104,26 @@ public static Object initializeSender(BObject senderClient, String connectionStr
      * @param message Input message record as a BMap
      * @return An error if failed to send the message
      */
-    public static Object send(BObject senderClient, BMap<BString, Object> message) {
-        try {
-            ServiceBusSenderClient sender = getSenderFromBObject(senderClient);
-            ServiceBusMessage messageToSend = constructMessage(message);
-            sender.sendMessage(messageToSend);
-            if (LOGGER.isDebugEnabled()) {
-                LOGGER.debug("Sent the message successfully. Message Id = " + messageToSend.getMessageId());
```

Comment:
Any particular reason to remove the existing debug logs from the code?

Reply:
These debug logs provide no additional value as they are not getting printed at any level. IMO these logs are unnecessary.